### PR TITLE
UI for getting bug with language filter without project

### PR DESCRIPTION
**Pull request by fiennyangeln**

### CODE REVIEW

**Comment by helfi92** [comment]
> "Feature/ui for bug without project"

What feature is that?

**Comment by fiennyangeln** [comment]
@helfi92 UI for going to the bug page with language filter without project

**Comment by fiennyangeln** [comment]
sorry  i just use my branch name directly! but idk if this UI is good as initially we dont have language selected, thus no query at first?

**Comment by helfi92** [comment]
Should we have a skill column similar to https://contribute.testpilot.firefox.com/tasks/ where we can a list of skills?

**Comment by fiennyangeln** [comment]
should we place it in a drawer so that we can stilll use the tasktable? yeah i think we can add the language icon too i just checked the mdi icons

**Comment by helfi92** [comment]
Yeah, a drawer would be good :)

File: src/views/Languages/index.jsx [hunk 1]
```diff
@@ -96,22 +140,25 @@ export default class Languages extends Component {
...
+          open={drawerOpen}
+          onClose={this.handleDrawerToggle}
+          classes={{
+            paper: classes.drawerPaper,
+          }}
+          ModalProps={{
+            keepMounted: true,
+          }}>
+          {drawer}
+        </Drawer>
```
Comment (helfi92) [inline src/views/Languages/index.jsx]: Picking a skill shows an empty table. Is this on purpose?

screen shot 2018-07-30 at 10 10 28 am

Comment (fiennyangeln) [reply src/views/Languages/index.jsx]: yes it is supposed to be handled by other PR #163

**Pull request closed without merging**